gensim/corpora/wikicorpus.py

```diff
@@ -214,9 +214,9 @@ def extract_pages(f, filter_namespaces=False):
             title = elem.find(title_path).text
             text = elem.find(text_path).text
 
-            ns = elem.find(ns_path).text
-            if filter_namespaces and ns not in filter_namespaces:
-                text = None
+            # ns = elem.find(ns_path).text
+            # if filter_namespaces and ns not in filter_namespaces:
+            #     text = None
```

Comment:
When I try to deal with old wiki dump using WikiCorpus, following error was returned.
![comment_out](https://cloud.githubusercontent.com/assets/6284510/18373437/ee425dbc-767e-11e6-80bc-5852eddc01bf.png)

To merge this feature, other fixing is as follows.
```
try:
    ns = elem.find(ns_path).text
    if filter_namespaces and ns not in filter_namespaces:
            text = None
except AttributeError:
    pass
```